openapi-diff/src/modeler/AutoRest.Swagger/ComparisonMessage.cs

```diff
@@ -15,35 +19,56 @@ public class ComparisonMessage
     {
         public static string DocBaseUrl = "https://github.com/Azure/openapi-diff/tree/master/docs/rules/";
 
-        public ComparisonMessage(MessageTemplate template, FileObjectPath path, Category severity, params object[] formatArguments)
+        public ComparisonMessage(
+            MessageTemplate template,
+            ObjectPath path,
+            IParsedJson old,
+            IParsedJson @new,
```

Comment:
why '@"?

Reply:
new is a keyword. We use `@` to say C# compiler that this is an identifier. Anyway, I renamed to `newDoc`